account_consolidation/wizard/consolidation_base.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2011-2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, _, api
+from odoo.exceptions import UserError
+
+
+class AccountConsolidationBase(models.AbstractModel):
+    _name = 'account.consolidation.base'
+    _description = 'Common consolidation wizard. Intended to be inherited'
+
+    @api.model
+    def _default_company(self):
+        return self.env['res.company']._company_default_get()
+
+    @api.model
+    def _default_subsidiaries(self):
```

Comment:
these 2 methods are still needed? Seems not. If not, remove also `_default_company` (as per my initial comment on default company)